### PR TITLE
add rules and vulns updates

### PR DESCRIPTION
**Pull request by rbren**

### CODE REVIEW

File: docs/configure/policy/rules.md [hunk 1]
```diff
@@ -36,12 +36,17 @@ The following fields can be edited:
...
-### Sending Slack Notifications
+
+## Integrations
+### Slack Notifications
 If you have attached a Slack installation to your organization, you can use the
 `sendSlackNotification` function to send messages. You can pick which channel
 to send to, or send via a webhook URL. You can also customize the message body
 to add mentions etc.
 
+You can also utilize [Slack incoming webhooks](https://slack.com/help/articles/115005265063-Incoming-webhooks-for-Slack)
```
Comment (makoscafee) [inline docs/configure/policy/rules.md:47]: shouldn't this title be on top of the actual example that utilizes the incoming webhook?

File: docs/configure/policy/rules.md [hunk 2]
```diff
@@ -55,6 +60,37 @@ if (ActionItem.Severity >= DANGER_SEVERITY && ActionItem.IsNew) {
 }
 ```
 
+```js
+if (ActionItem.Severity >= DANGER_SEVERITY && ActionItem.IsNew) {
+  sendSlackNotification(
```
Comment (makoscafee) [inline docs/configure/policy/rules.md:65]: Also mind adding `sendSlackNotification` arguments information like how you did for `createTicket`?

File: docs/run/agent/vulnerabilities.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Conatiner Vulnerabilities
```
Comment (makoscafee) [inline docs/run/agent/vulnerabilities.md:1]:
```suggestion
# Container Vulnerabilities
```
Typo 😄

**Review by makoscafee — APPROVED** [review]
LGTM. Just few questions/suggestions

File: docs/run/agent/vulnerabilities.md (hunk 3 above)
Comment (rbren) [reply docs/run/agent/vulnerabilities.md]: Thanks!

File: docs/configure/policy/rules.md (hunk 2 above)
Comment (rbren) [reply docs/configure/policy/rules.md:65]: Good call!

**Pull request merged**